sdk/core/core-client-rest/src/sendRequest.ts

```diff
@@ -114,7 +117,7 @@ function buildPipelineRequest(
   options: InternalRequestParameters = {},
 ): PipelineRequest {
   const requestContentType = getRequestContentType(options);
-  const { body, multipartBody } = getRequestBody(options.body, requestContentType);
+  const { body, multipartBody } = getRequestBody(options.body, requestContentType, options);
```

Comment:
Do you think we can have getRequestBody take only options? And inside it extract body and the requestContentType?